include/osquery/query.h

```diff
@@ -300,6 +304,14 @@ struct ScheduledQuery : private only_movable {
   ScheduledQuery(ScheduledQuery&&) = default;
   ScheduledQuery& operator=(ScheduledQuery&&) = default;
 
+  inline bool isSnapshotQuery() const {
+    return options.count("snapshot") && options.at("snapshot");
+  }
+
+  inline bool shouldClearRemovedRows() const {
+    return options.count("removed") && !options.at("removed");
+  }
```

Comment:
Would you like to create a tests for `shouldClearRemovedRows` and for `isSnapshotQuery` methods?